sql/moz-fx-data-marketing-prod/ga_derived/www_site_events_metrics_v2/query.sql

```diff
@@ -0,0 +1,55 @@
+-- Query for ga_derived.www_site_events_metrics_v2
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+SELECT
+  date,
+  event_name AS event_category,
+  event_name AS event_action,
+  event_name AS event_label,
+  page_path,
+  page_path_level1 AS locale,
+  page_level_1,
+  page_level_2,
+  page_level_3,
+  page_level_4,
+  page_level_5,
+  page_name,
+  device_category,
+  operating_system,
+  `language`,
+  browser,
+  country,
+  source,
+  medium,
+  campaign,
+  ad_content,
+  COUNT(*) AS total_events,
+  COUNT(DISTINCT visit_identifier) AS unique_events,
+FROM
+  `moz-fx-data-marketing-prod.ga_derived.www_site_hits_v2`
+WHERE
+  date = @submission_date
+  AND hit_type = 'EVENT'
```

Comment:
The only query type I see from 2023-01-01 is EVENT, I wonder if the filter is required?

Reply:
There are 2 types in the hits table, PAGE & EVENT 
<img width="550" alt="Screenshot 2024-01-29 at 10 46 37 AM" src="https://github.com/mozilla/bigquery-etl/assets/153020235/98811b72-781e-4a55-ac1c-c1475f2a4745">

Reply:
There you go, I only got one checking all 2023.